helpers/testing/runtime-env.ts

```
@@ -1,10 +1,12 @@
+import { BunRuntime } from '@effect/platform-bun';
+import { Effect } from 'effect';
 import path from 'node:path';
 
 const DEFAULT_APP_HOST_PORT = 4_200;
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)